src/org/gk/slicing/SlicingEngine.java

```diff
@@ -1318,14 +1605,14 @@ public static void main(String[] args) {
                 defaultPersonId = getInput("Enter the DB_ID for the default person to create InstanceEdit:");
             }
             MySQLAdaptor sourceDBA = new MySQLAdaptor(dbHost,
-                                                      dbName,
+                                                      DbName,
```

Comment:
Use dbName

Reply:
Fixed in [5d0061c](https://github.com/reactome/CuratorTool/pull/20/commits/5d0061cf2e6db424efa382db7c7d1255250b379b).